packs/feats/thoughtform-summoning.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You can create a prototypical creature from the depths of your imagination instead of calling one from elsewhere. Whenever you summon a creature, you can choose to instead create a psychic construct of that creature. This grants it resistance to physical damage equal to half its level, weakness 5 to force and mental damage, and the ability to pass through enemies' spaces without needing to @UUID[Compendium.pf2e.actionspf2e.Item.Tumble Through] (though it can't end its turn in an occupied space).</p>"
+            "value": "<p>You can create a prototypical creature from the depths of your imagination instead of calling one from elsewhere. Whenever you summon a creature, you can choose to instead create a psychic construct of that creature. This grants it resistance to physical damage equal to half its level, weakness 5 to force and mental damage, and the ability to pass through enemies' spaces without needing to @UUID[Compendium.pf2e.actionspf2e.Item.Tumble Through] (though it can't end its turn in an occupied space).</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Thoughtform Summoning]{Effect: Thoughtform Summoning}</p>"
```

Comment:
Was this extracted? I would have expected the label to be stripped.

Reply:
The extraction had come out wrong, I think because I extracted the feat before the effect by accident, and I tried to fix it manually, which was a bad idea. I think I did the extraction properly now.

Reply:
If you have things where they link to each other or you extract in the wrong order just running the extract again should fix the links.